enatega-multivendor-admin/public/index.html

```diff
@@ -32,6 +32,9 @@
 <link href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;900&display=swap" rel="stylesheet">
   <title>Enatega Multivendor Dashboard</title>
   <style>
+    body, html, #root {
```

Comment:
It is generally not recommended to apply styles directly in index.js for large scale application. The change can affect other app pages. So, its better to apply styles to the selected container only.

Reply:
@usama-sattar These styles are global and can be apply globally. These wouldn't affect other app pages because these are being used as a root styles.

These styles are only setting full body width that's it